README.md

```diff
@@ -47,6 +47,12 @@ require 'fulfillment-outbound-api-model'
       Rails.cache.write("SPAPI-TOKEN-#{access_token_key}", token[:access_token], expires_in: token[:expires_in] - 60)
     end
     config.get_access_token = -> (access_token_key) { Rails.cache.read("SPAPI-TOKEN-#{access_token_key}") }
+
+    # optional lambdas for caching grantless LWA access token instead of requesting it each time, e.g.:
+    config.save_grantless_access_token = -> (access_token_key, token) do
```

Comment:
thanks for including the readme update but it looks like we won't even need one it should "just work":

the lambdas already take an "access_token_key" which is the cache key to save the token under, let's use the same lambdas just vary the key similar to what i did for RDT https://github.com/ericcj/amz_sp_api/pull/55/files#diff-e2f2fe4083a87c67a20260d6f13385b63f2b64f0cabdcdf4e2f332fa754fbb5bR16

that way we can also remove the indirection of using public_send as well